spec/features/orders_form_spec.rb

```diff
@@ -19,6 +19,16 @@
     )
   end
 
+  describe "belongs_to relationships" do 
+    it 'uses the stored value as the selected value' do 
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.<br>Trailing whitespace detected.